app/controllers/eligibility_interface/countries_controller.rb

```diff
@@ -6,11 +6,20 @@ def new
       @country_form = CountryForm.new
     end
 
+    def next_url
+      {
+        eligible: url_helpers.eligibility_interface_qualifications_path,
+        ineligible: url_helpers.eligibility_interface_ineligible_path,
+        legacy: url_helpers.eligibility_interface_eligible_path,
+        region: url_helpers.eligibility_interface_region_path
+      }.fetch(eligibility_check.country_eligibility_status)
+    end
```

Comment:
```suggestion
      {
        eligible: eligibility_interface_qualifications_path,
        ineligible: eligibility_interface_ineligible_path,
        legacy: eligibility_interface_eligible_path,
        region: eligibility_interface_region_path
      }.fetch(eligibility_check.country_eligibility_status)
```

Just had a thought actually, I think this will work without needing a reference to `url_helpers`, as Rails controllers automatically include all the routing methods.